qiskit_aer/noise/__init__.py

```diff
@@ -26,30 +26,30 @@
 
 The :class:`NoiseModel` class is used to represent noise model for the
 :class:`~qiskit_aer.QasmSimulator`. It can be used to construct
-custom noise models for simulator, to to automatically generate a basic
-device noise model for an IBMQ backend.
+custom noise models for simulator, to automatically generate a basic
+device noise model for an `provider.fake_provider` backend.
 
 
 Device Noise Models
 -------------------
 
 A simplified approximate :class:`NoiseModel` can be generated automatically
-from the properties of real device backends from the IBMQ provider using the
-:meth:`NoiseModel.from_backend` method. See the method documentation for
-details.
+from the real noise data for an IBM Quantum device using the data stored in 
+`provider.fake_provider` using the :meth:`NoiseModel.from_backend` method. 
+See the method documentation for details.
```

Comment:
`FakeBackend` looks reasonable in the following example. However, Aer still supports generation of noise model from real backends for users to acquire the latest noise data. Can you add explanation to introduce noise data from real device backends?

Reply:
Yes, in the next commit I add both possibilities